rekall-core/rekall/plugins/overlays/windows/common.py

```diff
@@ -1291,13 +1292,21 @@ def file_name_with_drive(self, vm=None):
                 if name in drive_letter_device_map:
                     name = drive_letter_device_map.get(name)
 
-        filename = self.FileName.v(vm=vm)
+        filename = self.get_filename(vm=vm)
         if filename:
             name += utils.SmartUnicode(filename)
 
         return name
 
 
+    def get_filename(self, vm=None):
+        filename = self.FileName.v(vm=vm)
+ 
+        if (not filename or filename == "\x00") and self.FileName.Length > 0:
+            filename = "<Filename has probably been paged out>"
```

Comment:
On windows filenames with < or > are invalid. We probably need a better name than this or infact to sanitize the filename properly.